src/cluster.c

```diff
@@ -4440,6 +4440,110 @@ void restoreCommand(client *c) {
     server.dirty++;
 }
 
+void prepareRollback(struct redisCommand *cmd, client *c) {
```

Comment:
Not sure where to move. Just wondered if this was a thoughtful decision or just happened to be there for no reason.